src/go-contactless.html

```diff
@@ -130,20 +118,57 @@ <h3>Transport for London</h3>
       </picture>
       <h3>MTA New York City Transit</h3>
       <p>
-        On the first day that MTA launched contactless fares—at just 16 subway stations—nearly 6,000 riders tapped to
-        pay: 80% with a smart device and 20% with a bank card.
+        On the first day that MTA launched contactless fares—at just 16 subway stations—nearly 6,000 riders tapped to pay: 80%
+        with a smart device and 20% with a bank card.
       </p>
     </div>
   </div>
 </section>
 
-<hr>
-
-<section class="go-contactless-section pb-0 text-center" id="call-to-action">
+<section class="go-contactless-section go-contactless-timeline-container bg-dark-tan" id="call-to-action">
   <header>
-    <h2 class="mt-0">Ready to tap into the future of fare payment?</h2>
+    <h2 class="mt-0 mb-4 pb-3 text-center">Ready to tap into the future of fare payment?</h2>
   </header>
-  <section>
-    <a class="btn btn-outline-dark" href="/contracts">Get Started</a>
-  </section>
+  <div class="row mt-4 pt-3 d-flex flex-md-row">
+    <div class="procurement-image col-md-4 col-lg-4 text-center">
+      <img
+        alt="A smart phone and credit card being tapped at a bus fare device"
+        class="img-fluid contactless-cta-image"
+        src="/uploads/go-contactless-illo-payment-types.png"
+      />
+    </div>
+    <div class="procurement-timeline col-md-8 col-lg-8">
+      <ol class="timeline">
+        <li>
+          <div class="timeline-item">
+            <h3>Prepare</h3>
+            <p>Cal-ITP can help you prepare to start accepting contactless bank cards and smart devices’ digital wallets.</p>
+          </div>
+        </li>
+        <li>
+          <div class="timeline-item">
+            <h3>Purchase</h3>
+            <p>
+              The State of California has Master Service Agreements (MSAs) available to help you modernize your transit system.
+            </p>
+          </div>
+        </li>
+        <li>
+          <div class="timeline-item">
+            <h3>Implement</h3>
+            <p>Cal-ITP can provide the technical assistance you need to implement contactless payments across your fleet.</p>
+          </div>
+        </li>
+      </ol>
+    </div>
+    <div class="procurement-button col-md-12">
+      <div class="py-4 py-md-0 my-4 my-md-0 mx-auto text-center">
+        <h4 class="mt-4 pt-3 mb-3 d-md-block d-none w-50 mx-auto">
+          If you’d like to learn more about how to go contactless, please review the step-by-step guide or contact us for support.
```

Comment:
Since we're hiding the "View Guide" button [for now](https://github.com/cal-itp/mobility-marketplace/pull/386#issuecomment-1301463089), I think we should also hide `review the step-by-step guide or` from this sentence.